modules/terrain/docs/api-reference/quantized-mesh-loader.md

```diff
@@ -39,6 +39,7 @@ const data = await load(url, QuantizedMeshLoader, options);
 | Option                  | Type            | Default        | Description                                                                     |
 | ----------------------- | --------------- | -------------- | ------------------------------------------------------------------------------- |
 | `quantized-mesh.bounds` | `array<number>` | `[0, 0, 1, 1]` | Bounds of the image to fit x,y coordinates into. In `[minX, minY, maxX, maxY]`. |
+| `terrain.skirtHeight`   | `number`        | `null`         | If set, create the skirt for the tile with particular height                    |
```

Comment:
We should define what units this height is in. I assume it's in meters if the elevation values are in meters?